templates/_layouts/_footer.html

```diff
@@ -13,6 +13,9 @@
           <li class="p-inline-list__item">
             <a class="p-link--external" href="https://github.com/canonical-web-and-design/vanilla-framework/issues/new/choose">Report a bug</a>
           </li>
+          <li class="p-inline-list__item">
+            <a href="" class="js-revoke-cookie-manager">Manage your tracker settings</a>
```

Comment:
Would be good not to leave href empty.

```suggestion
            <a href="#tracker-settings" class="js-revoke-cookie-manager">Manage your tracker settings</a>
```

Reply:
There is no where for this to default. If JS is switched off tracking won't work as they are injected via scripts.